src/aa_device_periods.cu

```diff
@@ -799,6 +800,15 @@ namespace astroaccelerate {
     (*compute_time) = (*compute_time) + timer.Elapsed();
     //---------<
 	
+	//---------> Spectrum whitening
+	timer.Start();
+	cudaStream_t stream; stream = NULL;
+	spectrum_whitening_SGP1(d_frequency_power, (t_nTimesamples>>1), t_nDMs_per_batch, stream);
```

Comment:
Just to be sure, the acknowledge here is to have the streams at some point and thus we are creating streams as default NULL?

Reply:
Yes, this is for the possibility of including streams in the future. For the kernel to be launched in the default stream we can use NULL.

Reply:
OK